core/cache/redis.go

```
@@ -0,0 +1,47 @@
+	dsn := fmt.Sprintf("redis://:%s@%s:%d/%d",
+		config.Redis.Password,
+		config.Redis.Host,
+		config.Redis.Port,
+		config.Redis.Db)
+	opt, err := redis.ParseURL(dsn)
+	if err != nil {
+		return nil, fmt.Errorf("redis 解析dsn失败: %v", err)
+	}
+	poolSize := config.Redis.PoolSize
+	if poolSize == 0 {
+		poolSize = 10
```

golangci-lint (1.64.8)

ineffectual assignment to poolSize

(ineffassign)